mk-modules.nix

```
@@ -0,0 +1,86 @@
+{ flakeInputs, mkModulesOpts }: let
+  lib = flakeInputs.nixpkgs.lib;
+
+  systems = mkModulesOpts.systems or [
+    "aarch64-darwin"
+    "aarch64-linux"
+    "x86_64-darwin"
+    "x86_64-linux"
+  ];
+
+  flakeSchema.options = {
```

```suggestion
  flakeSchemaModule.options = {
```
Maybe? To make more explicit that this is a module?